source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/calculation/CalculationJobType.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright the State of the Netherlands
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+package nl.overheid.aerius.shared.domain.calculation;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Locale;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import jsinterop.annotations.JsProperty;
+
+import nl.overheid.aerius.shared.domain.scenario.SituationType;
+
+/**
+ * type of Calculation Job
+ */
+public enum CalculationJobType {
+
+  /**
+   * Exactly 1 Project scenario
+   * Optional: 1 off site reduction
+   * Optional: 1 Reference
+   * NO other scenario types
+   */
+  PROCESS_CONTRIBUTION(new HashSet<>(Arrays.asList(SituationType.PROPOSED)),
```

Comment:
`EnumSet` doesn't work?

Reply:
I thought `EnumSet` isn't supported in GWT